installer/bundle/bundle_skel.sh

```diff
@@ -307,6 +448,42 @@ do
             cleanup_and_exit 0
             ;;
 
+        --version)
+            echo "Version: `getVersionNumber $OMS_PKG omsagent-`"
+            exit 0
+            ;;
+
+        --version-check)
+            printf '%-15s%-15s%-15s%-15s\n\n' Package Installed Available Install?
+
+            # omi
+            versionInstalled=`getInstalledVersion omi`
+            versionAvailable=`getVersionNumber $OMI_PKG omi-`
+            if shouldInstall_omi; then shouldInstall="Yes"; else shouldInstall="No"; fi
+            printf '%-15s%-15s%-15s%-15s\n' omi $versionInstalled $versionAvailable $shouldInstall
+
+            # scx
+            versionInstalled=`getInstalledVersion scx`
+            versionAvailable=`getVersionNumber $SCX_PKG scx-cimprov-`
+            if shouldInstall_scx; then shouldInstall="Yes"; else shouldInstall="No"; fi
+            printf '%-15s%-15s%-15s%-15s\n' scx $versionInstalled $versionAvailable $shouldInstall
+
+            # OMS agent itself
+            versionInstalled=`getInstalledVersion omsagent`
+            versionAvailable=`getVersionNumber $OMS_PKG omsagent-`
+            if shouldInstall_omsagent; then shouldInstall="Yes"; else shouldInstall="No"; fi
+            printf '%-15s%-15s%-15s%-15s\n' omsagent $versionInstalled $versionAvailable $shouldInstall
+
+            # omsconfig
+            versionInstalled=`getInstalledVersion omsconfig`
+            versionAvailable=`getVersionNumber $DSC_PKG omsconfig-`
+            if ! ptyon_ctypes_installed 1> /dev/null 2> /dev/null; then ctypes_text=" (No ctypes)"; fi
```

Comment:
I only tested this in the failed case and, since stderr/stdout was redirected, I missed this in my testing. Thanks for the good catch!